phpseclib/Net/Ssh2/MessageType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace phpseclib3\Net\Ssh2;
```

Comment:
Could you rename it to SSH2 instead of Ssh2?

Both SSH2 and SFTP are all caps as is the newly added SFTP directory. By that same reasoning it stands to reason that it ought to be SSH2 instead of Ssh2.

Thanks!

Reply:
Done